tutorials/cp-cf-understand-application-lifecycle/cp-cf-understand-application-lifecycle.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Understand the Cloud Foundry Application Lifecycle for Buildpack Applications
+description: Understand the application lifecycle in Cloud Foundry and what happens when you cf push.
+auto_validation: true
+time: 15
+tags: [ tutorial>beginner, software-product>sap-btp--cloud-foundry-environment]
+primary_tag: software-product>sap-btp--cloud-foundry-environment
+author_name: Beyhan Veli
+author_profile: https://github.com/beyhan
+---
+
+## Prerequisites
+ - **Tutorials** [Fundamentals of the SAP BTP, Cloud Foundry environment](cp-cf-fundamentals)
+ - **Tutorials** [Install the Cloud Foundry Command Line Interface (CLI)](cp-cf-download-cli)
+
+
+## Details
+### You will learn
+  - What happens when you CF Push
+  - The lifecycle of buildpack applications
+  - Differences between restarting, re-staging, and re-deploying
+  - How Cloud Foundry handles crashes and shutdowns, and the process of evacuation
+
+---
+[ACCORDION-BEGIN [Step 1: ](CF Push)]
```

Comment:
Please use an action verb for each step.  For example, this one might be "Push Cloud Foundry" or whatever the proper verb is.  Remember that we are asking users to do things in a tutorial.

Reply:
This feedback has been applied.